pkg/controller/gitopsservice/gitopsservice_controller_test.go

```diff
@@ -26,4 +26,26 @@ func TestImageFromEnvVariable(t *testing.T) {
 			t.Errorf("Image mismatch: got %s, want %s", got, backendImage)
 		}
 	})
+
+	t.Run("Kam Image present as env variable", func(t *testing.T) {
+		image := "quay.io/org/test"
+		os.Setenv(kamImageEnvName, image)
+		defer os.Unsetenv(kamImageEnvName)
```

Comment:
I think this should be `cliImageEnvName`?

The tests are erroring without it.

Reply:
Good catch, didn't test after the last minute renaming.